meta-refkit-core/recipes-kernel/linux/refkit-kernel-cache/tpm2.cfg

```diff
@@ -0,0 +1,15 @@
+..........................................................................
+.                                WARNING
+.
+. This file is a kernel configuration fragment, and not a full kernel
+. configuration file.  The final kernel configuration is made up of
+. an assembly of processed fragments, each of which is designed to
+. capture a specific part of the final configuration (e.g. platform
+. configuration, feature configuration, and board specific hardware
+. configuration).  For more information on kernel configuration, please
+. consult the product documentation.
+.
+..........................................................................
+
+CONFIG_TCG_TPM=y
+CONFIG_TCG_CRB=m
```

Comment:
Looks like we need ```CONFIG_TCG_CRB=y``` or have the module installed and loaded for fTPM to get the device interface available in initramfs.